test/contract/coverity.e2e.test.ts

```diff
@@ -139,9 +125,9 @@ export function mockCoverityParamsExcept(coverityConstants: string[]) {
 }
 
 export function setAllMocks() {
+  let coverity: string[] = []
   jest.spyOn(configVariables, 'getWorkSpaceDirectory').mockReturnValue(__dirname)
-  jest.spyOn(validator, 'validatePolarisInputs').mockReturnValueOnce(true)
-
+  jest.spyOn(validator, 'validateCoverityInputs').mockReturnValueOnce(coverity)
```

Comment:
any reason for changing polaris to coverity?

Reply:
setting respective mocks for the validate methods.